docs/src/reference/remote_inputs.rst

```diff
@@ -170,3 +170,8 @@ Compressed vs uncompressed starting points
 ------------------------------------------
 
 The workflow supports compressed metadata and sequences for any input stage. Files may be compressed using Zstandard (``.zst``), xz (``.xz``), or gzip (``.gz``) compression.
+
+Data origins
+------------------------------------------
+
+The data for our open dataset comes from [NCBI Genbank](https://www.ncbi.nlm.nih.gov/) (via API), and the [Robert Koch Institute (RKI)](https://github.com/robert-koch-institut/SARS-CoV-2-Sequenzdaten_aus_Deutschland). Some UK metadata is augmented with data available from [COG-UK](https://www.cogconsortium.uk/priority-areas/data-linkage-analysis/public-data-analysis/) (via CLIMB).
```

Comment:
This is Markdown syntax for links, but the document is rST.

Reply:
([Preview of the page](https://nextstrain--1072.org.readthedocs.build/projects/ncov/en/1072/reference/remote_inputs.html) shows how it doesn't render right.)

Reply:
I've tried to amend this to match the URL style earlier in the doc - I'd like to check the preview but it doesn't seem to have updated - is there a way to regenerate it?

Reply:
Ah - in the 'rich diff' on the changed files page, I seem to be able to see that the URLs are displaying properly now! But never hurts to have a second check...

Reply:
The preview page does update automatically, but it might have been delayed…

Reply:
Ok, in that case I think this is showing up correctly now!